src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -926,6 +926,11 @@ static ucs_status_t uct_ib_mlx5dv_md_open(struct ibv_device *ibv_device,
         goto err_free;
     }
 
+#if HAVE_STRUCT_IBV_DEVICE_ATTR_EX_PCI_ATOMIC_CAPS
```

Comment:
should we move this after line 921, inside `if (IBV_EXP_HAVE_ATOMIC_HCA(&dev->dev_attr)` ?